ffi/src/lib.rs

```diff
@@ -155,6 +211,16 @@ impl From<Box<[u8]>> for Value {
     }
 }
 
+impl From<String> for Value {
```

Comment:
Use `From<&str>` (not `From<str>`). You don't need an owned string to convert it to a value, especially since you'll be coping the data out of it anyway.

See See https://doc.rust-lang.org/book/ch20-03-advanced-types.html#dynamically-sized-types-and-the-sized-trait for more info on unsized types, and in particular `str`.